src/NuGet.Clients/PackageManagement.UI/Resources.resx

```diff
@@ -511,4 +511,7 @@ Packages installed into Website projects will not be restored during build. Cons
     <value>not installed</value>
     <comment>In the solution package manager, if the select package is not installed, then this message is shown.</comment>
   </data>
+  <data name="Text_UserCancelled" xml:space="preserve">
+    <value>User cancelled operation</value>
```

Comment:
add a dot at the end of the message, And change to `operation canceled by user.`